tasks/add-fbc-contribution/README.md

```diff
@@ -14,6 +14,10 @@ Task to create a internalrequest to add fbc contributions to index images
 | targetIndex    | targetIndex value updated by update-ocp-tag task                                          | No       | -                    |
 | resultsDirPath | Path to results directory in the data workspace                                           | No       | -                    |
 
+## Changesin 3.3.2
+* Removed references to data parameters `iibServiceConfigSecret`, `iibServiceAccountSecret` and `iibOverwriteFromIndexCredential` as
```

Comment:
Isn't only ServiceConfigSecret removed? I am not seeing ServiceAccountSecret references, but maybe I am missing it

Reply:
You are correct. It was removed already and I blindly copied the list from the ticket. Fixed.